src/style-value.js

```diff
@@ -68,7 +68,7 @@
     return styleValueArray;
   };
 
-  internal.StyleValue = StyleValue;
+  scope.StyleValue = StyleValue;
```

Comment:
If we're exposing this here, we should probably throw an exception if someone tries to write `new StyleValue()` too.